packages/dapp/src/providers/WalletProvider.tsx

```diff
@@ -101,14 +105,12 @@ export const WalletProvider: React.FC<PropsWithChildren<{}>> = ({
       await liFiWalletManagement.connect(wallet);
       trackEvent({
         action: TrackingAction.ConnectWallet,
+        category: TrackingCategory.Wallet,
+        label: 'connect-wallet',
```

Comment:
@Addminus Upfront, I could bet you´re not a big fan of it, as we recently removed the labels and category for all of those within WalletProvider. Cookie3-events rather use a label and like to have a category as required props.
Since we have a very clean setup in place where only those few are lacking a label and a category, I found it reasonable to add it back here. 

To be fair: It´s not stated in their documentation but going from their Typescript definitions and also from what is being sent (analyzing the network tab), that "name" (a.k.a our label) seems to be meant for something.
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/1eda4b4f-7bee-4b89-a418-46c92e48afb0)

Wdyt?

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/757c787a-26b1-4d60-8df3-b67179af9f15)

Reply:
we can add the label as param and only use it in the cookie 3 event function. as with name: as far as i understand its the name of the action the user did. use the Tracking action keys for it

Reply:
To be on the same page:
1.) I added the label as a required field to all events. This way, we can also make use of that number it in the cookie3 event. 

2.) You were saying (?): "name" attribute should use our action-value and "action" should be used for what we got as a label? I might misinterpret here, but from that I would rather stick with what we´ve got. Both are pretty descriptive and it should be very readable anyways.